src/FluentContracts/FluentContracts.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
     <!--General-->
     <PropertyGroup>
-        <TargetFramework>net6.0</TargetFramework>
+        <TargetFramework>net8.0</TargetFramework>
```

Comment:
I am intentionally not putting latest .NET version here, so that the package can be targeted by a wider range of users, who are still not using the latest version. I don't need any features of `net8.0` for now, so let's keep it at version `6.0`

Reply:
Ok. I understand.

Reply:
You still need to push a commit, that reverts the version to `net6.0`, in order for me to approve it.